packages/position/src/react/index.js

```diff
@@ -59,7 +59,7 @@ const Position = React.forwardRef((props, forwardedRef) => {
   const showRef = React.useRef()
   const showEl = React.cloneElement(props.show, {
     ref: showRef,
-    style: { ...child.props.style, ...style }
+    style: { ...props.show.props.style, ...style }
```

Comment:
I believe that the fix should be transfer `show.props.style` to the cloned `props.show` element instead of removing it.

[cloneElement](https://reactjs.org/docs/react-api.html#cloneelement) does shallow merge.  But, remember, shallow is at the first level properties, where the style property is.  It doesn't merge any deeper, including the style object properties.  I added a story to verify this fact and need to keep this.  Please verify and see what you think.